frame/composable-traits/src/staking/mod.rs

```diff
@@ -131,7 +131,7 @@ pub struct Stake<RewardPoolId, Balance, Rewards> {
 	pub share: Balance,
 
 	/// Reduced rewards by asset for the position (d_n)
-	reductions: Rewards,
+	pub reductions: Rewards,
```

Comment:
```suggestion
	pub reductions: Reductions,
```

Let's rename the generic parameter so that it makes sense